subscription_platform/dashboards/monitor_saasboard__revenue.dashboard.lookml

```diff
@@ -0,0 +1,661 @@
+      monthly_active_logical_subscriptions.logical_subscription_count]
+    fill_fields: [monthly_active_logical_subscriptions.month_month]
+    filters:
+      subscription_services.id: Monitor
+      countries.region_name: ''
+      countries.name: ''
+      current_subscription_state.has_fraudulent_charges: 'No'
+      current_subscription_state.has_refunds: ''
+      monthly_active_logical_subscriptions.was_active_at_month_end: 'Yes'
+    sorts: [monthly_active_logical_subscriptions.logical_subscription_count]
+    limit: 500
+    column_limit: 50
+    dynamic_fields:
+    - category: table_calculation
+      expression: "${monthly_active_logical_subscriptions.total_annual_recurring_revenue_usd}\
+        \ / ${monthly_active_logical_subscriptions.logical_subscription_count}"
+      label: ARPU (ARR/Active Subs)
+      value_format:
+      value_format_name: decimal_1
+      _kind_hint: measure
+      table_calculation: arpu_arractive_subs
+      _type_hint: number
+    x_axis_gridlines: false
+    y_axis_gridlines: true
+    show_view_names: false
+    show_y_axis_labels: true
+    show_y_axis_ticks: true
+    y_axis_tick_density: default
+    y_axis_tick_density_custom: 5
+    show_x_axis_label: true
+    show_x_axis_ticks: true
+    y_axis_scale_mode: linear
+    x_axis_reversed: false
+    y_axis_reversed: false
+    plot_size_by_field: false
+    trellis: ''
+    stacking: ''
+    limit_displayed_rows: false
+    legend_position: center
+    point_style: circle
+    show_value_labels: true
+    label_density: 25
+    x_axis_scale: auto
+    y_axis_combined: true
+    ordering: none
+    show_null_labels: false
+    show_totals_labels: true
+    show_silhouette: false
+    totals_color: "#808080"
+    y_axes: [{label: ARPU in $, orientation: left, series: [{axisId: arpu_arractive_subs,
+            id: arpu_arractive_subs, name: ARPU (ARR/Active Subs)}], showLabels: true,
+        showValues: true, valueFormat: "$0", unpinAxis: false, tickDensity: default,
+        tickDensityCustom: 5, type: linear}]
+    x_axis_label: Month
+    x_axis_zoom: true
+    y_axis_zoom: true
+    label_value_format: "$0"
+    series_types:
+      arpu_arractive_subs: line
+    series_colors:
+      monthly_active_logical_subscriptions.total_annual_recurring_revenue_usd: "#FF7139"
+    show_null_points: true
+    interpolation: linear
+    defaults_version: 1
+    hidden_fields: [monthly_active_logical_subscriptions.logical_subscription_count,
+      monthly_active_logical_subscriptions.total_annual_recurring_revenue_usd]
+    listen:
+      Payment Provider: monthly_active_logical_subscriptions.subscription__payment_provider
+      Active Date: monthly_active_logical_subscriptions.month_month
+      Plan Interval: monthly_active_logical_subscriptions.subscription__plan_interval
+      Plan: monthly_active_logical_subscriptions.subscription__plan_summary
```

Comment:
- This chart needs to listen to all of the filters.
- Suggest moving the note about the calculation from the title to an info tool-tip.
- Suggest not filling missing dates.
- Suggest sorting by date, not count.
```suggestion
  - title: Average Revenue Per Unit
    name: Average Revenue Per Unit
    model: subscription_platform
    explore: monthly_active_logical_subscriptions
    type: looker_column
    fields: [monthly_active_logical_subscriptions.month_month, monthly_active_logical_subscriptions.total_annual_recurring_revenue_usd,
      monthly_active_logical_subscriptions.logical_subscription_count]
    filters:
      monthly_active_logical_subscriptions.was_active_at_month_end: 'Yes'
    sorts: [monthly_active_logical_subscriptions.month_month desc]
    limit: 500
    column_limit: 50
    dynamic_fields:
    - category: table_calculation
      expression: "${monthly_active_logical_subscriptions.total_annual_recurring_revenue_usd}\
        \ / ${monthly_active_logical_subscriptions.logical_subscription_count}"
      label: ARPU (ARR/Active Subs)
      value_format:
      value_format_name: decimal_1
      _kind_hint: measure
      table_calculation: arpu_arractive_subs
      _type_hint: number
    x_axis_gridlines: false
    y_axis_gridlines: true
    show_view_names: false
    show_y_axis_labels: true
    show_y_axis_ticks: true
    y_axis_tick_density: default
    y_axis_tick_density_custom: 5
    show_x_axis_label: true
    show_x_axis_ticks: true
    y_axis_scale_mode: linear
    x_axis_reversed: false
    y_axis_reversed: false
    plot_size_by_field: false
    trellis: ''
    stacking: ''
    limit_displayed_rows: false
    legend_position: center
    point_style: circle
    show_value_labels: true
    label_density: 25
    x_axis_scale: auto
    y_axis_combined: true
    ordering: none
    show_null_labels: false
    show_totals_labels: true
    show_silhouette: false
    totals_color: "#808080"
    y_axes: [{label: ARPU in $, orientation: left, series: [{axisId: arpu_arractive_subs,
            id: arpu_arractive_subs, name: ARPU (ARR/Active Subs)}], showLabels: true,
        showValues: true, valueFormat: "$0", unpinAxis: false, tickDensity: default,
        tickDensityCustom: 5, type: linear}]
    x_axis_label: Month
    x_axis_zoom: true
    y_axis_zoom: true
    label_value_format: "$0"
    series_types:
      arpu_arractive_subs: line
    series_colors:
      monthly_active_logical_subscriptions.total_annual_recurring_revenue_usd: "#FF7139"
    show_null_points: true
    interpolation: linear
    defaults_version: 1
    hidden_fields: [monthly_active_logical_subscriptions.logical_subscription_count,
      monthly_active_logical_subscriptions.total_annual_recurring_revenue_usd]
    note_state: collapsed
    note_display: hover
    note_text: ARPU = ARR / Active Subscriptions
    listen:
      Active Date: monthly_active_logical_subscriptions.month_month
      Payment Provider: monthly_active_logical_subscriptions.subscription__payment_provider
      Plan Interval: monthly_active_logical_subscriptions.subscription__plan_interval
      Plan: monthly_active_logical_subscriptions.subscription__plan_summary
      Region: countries.region_name
      Country: countries.name
      Has Fraudulent Charges (Yes / No): current_subscription_state.has_fraudulent_charges
      Has Refunds (Yes / No): current_subscription_state.has_refunds
      Service ID: subscription_services.id
```